src/oaklib/utilities/lexical/lexical_indexer.py

```diff
@@ -247,6 +247,7 @@ def lexical_index_to_sssom(
     ruleset: MappingRuleCollection = None,
     meta: Optional[Dict[str, t.Any]] = None,
     prefix_map: Union[None, Converter, t.Mapping[str, str]] = None,
+    extended_prefix_map: t.Iterable[Dict[str, t.Any]] = None,
```

Comment:
## `extended_prefix_map` param added
I also wanted it to have the same type signature as in `curies`, which is `LocationOr[Iterable[Union[Record, Dict[str, Any]]]] = None`. However, when I did `from curies.mapping_service.utils import Records`, I got `ModuleNotFoundError: No module named 'defusedxml'`. I definitely don't think it's worth installing an additional package in OAK simply for better / more consistent typing.

I suppose an alternative to adding a new param would simply to overload the `prefix_map` param even more, but I opted not because itis already overloaded. I actually don't like this param because it supports either a prefix map `Dict` or a `Converter` object, and I think that makes it a little unclear / poorly named.